common/app/views/fragments/inlineJSBlocking.scala.html

```diff
@@ -6,13 +6,14 @@
 
 @(page: model.Page)(implicit request: RequestHeader, context: model.ApplicationContext)
 
-<!--[if lt IE 9]>
-    <script src="@Static("javascripts/es5-html5.js")"></script>
-<![endif]-->
+@if(!mvt.PolyfillIO.isParticipating) {
+    <!--[if lt IE 9]>
+        <script src="@Static("javascripts/es5-html5.js")"></script>
```

Comment:
Ah, good catch. I didn't know `html5shiv` is bundled into this. So, we have to keep it for now.

But maybe interesting for your branch then:

```js
// bundle-shims.js

                require.resolve('es5-shim'),
                require.resolve('html5shiv'),
                path.resolve(
                    path.dirname(require.resolve('JSON2')),
                    'json2.js'
                ),
```

We could remove `require.resolve('es5-shim')` and `path.dirname(require.resolve('JSON2'))` then and add `JSON3` to pf.io aswell.